src/pages/instances/InstanceOverviewDeviceList.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { isRootDisk } from "util/instanceValidation";
+import { FormDevice } from "util/formDevices";
+import { LxdDeviceValue, LxdGPUDevice } from "types/device";
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  device: LxdDeviceValue;
+  instance: LxdInstance;
```

Comment:
since we only need the project, why not just pass `project: string` down as a prop instead of the entire instance? This is a general good practice in react, since prop changes causes components to re-render. Trying to limit the prop interface size is an optimisation.